lib/add_movie/add_movie_screen.dart

```diff
@@ -0,0 +1,206 @@
+}
+
+class _AddMovieScreenState extends State<AddMovieScreen> {
+  String? title;
+  String? description;
+
+  final _formKey = GlobalKey<FormState>();
+
+  _AddMovieScreenState();
+  AddMovieCubit get addCubit => BlocProvider.of<AddMovieCubit>(context);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocListener<AddMovieCubit, AddMovieState>(
+      listener: (context, state) {
+        if (state.status == AddMovieStatus.success) {
+          BlocProvider.of<MyMovieCubit>(context).data(
+            title!,
+            description!,
+          );
+        }
+      },
+      child: Scaffold(
+        backgroundColor: AppColors.darkBlue,
+        appBar: AppBar(
+          centerTitle: true,
+          title: const Text(_kTitle),
+        ),
+        body: BlocBuilder<AddMovieCubit, AddMovieState>(
+          builder: (context, state) {
+            return Form(
+              key: _formKey,
+              child: Center(
+                child: SingleChildScrollView(
+                  padding: const EdgeInsets.only(
+                    top: 10,
+                    left: 8.0,
+                    right: 8.0,
+                  ),
+                  child: Column(
+                    children: [
+                      _TitleMovie(
+                        onSaved: (value) {
+                          title = value;
+                        },
+                      ),
+                      const SizedBox(height: 20),
+                      _DescriptionMovie(
+                        onSaved: (value) {
+                          description = value;
+                        },
+                      ),
+                      const SizedBox(height: 20),
+                      FloatingActionButton(
+                        backgroundColor: AppColors.darkBlueBackground,
+                        child: const Icon(Icons.add),
+                        onPressed: _onPressedSave,
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+
+  void _onPressedSave() {
+    if (_formKey.currentState!.validate()) {
+      _formKey.currentState!.save();
+      addCubit.saveTitleDescription(title!, description!);
+      Navigator.pop(context);
+    }
+  }
+}
+
+class _TitleMovie extends StatelessWidget {
+  const _TitleMovie({
+    Key? key,
+    required this.onSaved,
+    this.title,
+  }) : super(key: key);
+
+  final ValueChanged<String?> onSaved;
+  final String? title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Theme(
+      data: Theme.of(context).copyWith(splashColor: Colors.transparent),
+      child: TextFormField(
+        validator: _validate,
+        onSaved: onSaved,
+        autofocus: false,
+        style: context.theme.textTheme.subtitle1!
+            .copyWith(color: AppColors.darkBlue),
+        decoration: InputDecoration(
```

Comment:
create class `_ExtendedInputDecoration` which will extend `InputDecoration ` and into `super` add all properties
and `title, hintText` should be property of _ExtendedInputDecoration class